src/group_theory/sylow.lean

```diff
@@ -91,6 +92,39 @@ begin
 end
 ... = _ : by simp; refl
 
+/--If a p-group acts on `α` and the cardinality of `α` is not a multiple
+  of `p` then the action has a fixed point. -/
+lemma exists_fixed_point_of_prime_not_dvd_card
```

Comment:
This should maybe be called
```suggestion
lemma nonempty_fixed_point_of_prime_not_dvd_card
```
Since the lemma statement does not have an exists